boards/common/nrf52/include/periph_conf_common.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_nrf52
+ * @{
+ *
+ * @file
+ * @brief       Common peripheral configuration for nRF52 based boards
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ */
+
+#ifndef PERIPH_CONF_COMMON_H
+#define PERIPH_CONF_COMMON_H
+
+#include "periph_cpu.h"
+#include "cfg_clock_32_1.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Timer configuration
+ * @{
+ */
+static const timer_conf_t timer_config[] = {
+    {
+        .dev      =  NRF_TIMER1,
+        .channels =  3,
+        .bitmode  = TIMER_BITMODE_BITMODE_32Bit,
+        .irqn     = TIMER1_IRQn
+    }
+};
```

Comment:
I was just thinking: if any board ever wants to do some alternative timer configuration, it would need not to include this header and re-define both `timer` and `rtt` config. How about we further split this file and call them `cfg_timer_default.h` and `cfg_rtt_default.h`?

Reply:
This sounds like a good idea.

Reply:
Done

Reply:
nice